src/plugins/console/server/lib/utils/index.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { URLSearchParams } from 'url';
+import { trimStart } from 'lodash';
+
+export const encodePath = (path: string) => {
```

Comment:
Could you extract this into a `encode_path.ts` file, and re-export `encodePath` from this file? Then could also rename `utils.test.ts` to `encode_path.test.ts`? I think this will make it easier to navigate the codebase by scanning files and being able to understand their role without having to open them individually. It will also scale better as we add more utils.